### PR TITLE
treehouses bluetoothid [number] (fixes #436)

### LINKED ISSUE #436: treehouses bluetoothid?
can we print out the 4 digit number which is added after the hostname

### PR DESCRIPTION
**Pull request by CalebProvost**

### CODE REVIEW

File: modules/bluetoothid.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  bid=$(cat /etc/bluetooth-id)
+  id=${bid}
+
+  #echo network name with bluetooth id
+  nname=$(uname -n)
+  name=${nname}
+
+  case "$1" in
+    "")
+      echo "Raspberry Pi's ID is: ${id}"
```
Comment (dogi) [inline modules/bluetoothid.sh]: the bluetooth id is ???

File: modules/bluetoothid.sh [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      ;;
+    "hostid")
+      echo "Raspberry Pi's network Host ID is: ${name}-${id}"
```
Comment (dogi) [inline modules/bluetoothid.sh]: same as above

File: modules/bluetoothid.sh [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      ;;
+  esac
+}
+
+function bluetoothid_help () {
+  echo ;
```
Comment (dogi) [inline modules/bluetoothid.sh]: help looks not too good

File: modules/help.sh [hunk 4]
```diff
@@ -22,6 +22,7 @@ function help_default {
   echo "          [password] [hotspotPassword]"
   echo "   container <none|docker|balena>            enables (and start) the desired container"
   echo "   bluetooth <on|off>                        switches between bluetooth hotspot mode / regular bluetooth and starts the service"
+  echo "   bluetoothid [hostid]                      displays 4 digit bluetooth id [displays as 'networkname-id']"
```
Comment (dogi) [inline modules/help.sh]: not sure about message in `[]`

File: README.md [hunk 5]
```diff
@@ -29,6 +29,7 @@ bridge <ESSID> <hotspotESSID>             configures the rpi to bridge the wlan
        [password] [hotspotPassword]
 container <none|docker|balena>            enables (and start) the desired container
 bluetooth <on|off>                        switches between bluetooth hotspot mode / regular bluetooth and starts the service
+bluetoothid [hostid]			  displays 4 digit bluetooth id [displays as 'networkname-id']
```
Comment (dogi) [inline README.md]: not sure about message in `[]`

File: modules/bluetoothid.sh (hunk 3 above)
Comment (vmnet8) [reply modules/bluetoothid.sh]: I agreed it .  
```
pi@pi:~/git_repo/cli $ ./cli.sh help bluetoothid 

        Usage: cli.sh bluetoothid [hostid]

                Displays Raspberry Pi's Bluetooth Host Number.
          Optionally displays Networkname and ID as recieved through pairing.

        Example:
/home/pi/git_repo/cli/modules/bluetoothid.sh: line 30:    cli.sh bluetoothid hostid: command not found

                treehouses-9012
```

File: modules/bluetoothid.sh (hunk 1 above)
Comment (CalebProvost) [reply modules/bluetoothid.sh]: `echo "Bluetooth ID digits: ${id}"`

File: modules/bluetoothid.sh [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+  nname=$(uname -n)  #get network name
+
+  case "$1" in
+    "")
+      echo "Bluetooth ID is: ${bid}"
+      ;;
+    "hostid")
+      echo "Bluetooth's Network Host ID is: ${nname}-${bid}"
+      ;;
+  esac
```
Comment (phamduchongan93) [inline modules/bluetoothid.sh]: *)
 echo "Error: please leave empty argument or type in hostid"
;;

Comment (CalebProvost) [reply modules/bluetoothid.sh]: I was thinking the same. Wasn't sure if it would be necessary.
@dogi, should this be added?

**Comment by dogi** [comment]
`treehouses help bluetoothid` is not up to code :P

**Pull request merged**